CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.0.1
+-----
+
+* Guzzle ^6.5 allowed to make LTI core lib compatible with TAO CGEN
```

Comment:
We try to prefix each modification with
`Added` `Moved` `Removed` `Fixed` `Updated`
Can we modify it (like `* Updated Guzzle making ^6.5 allowed to have LTI core lib compatible with TAO CGEN`) please? :slightly_smiling_face:

Reply:
fixed, thank you!